includes/query/SMW_QueryParser.php

```diff
@@ -139,6 +139,31 @@ protected function getSubqueryDescription( &$setNS ) {
 			$setsubNS = false;
 
 			switch ( $chunk ) {
+				case '!':
+					$count = 0;
+					while ( $chunk == '!' ) {
+						$count ++;
+						$chunk = $this->readChunk();
+					}
+					if ( $chunk == '<q>' ) {
+						$this->pushDelimiter( '</q>' );
+						$ld = $this->getSubqueryDescription( $setsubNS );
+						if ( $ld instanceof SMWDisjunction )
+						{
+							foreach ( $ld->getDescriptions() as $desc )
+							{
+								$conjunction = $this->addDescription( $conjunction, $desc, SMW_NEGATION_QUERY );
+							}
+						} else {
+							$conjunction = $this->addDescription( $conjunction, $ld, SMW_NEGATION_QUERY );
+						}
+					} else {
+						$ld = $this->getLinkDescription( $setsubNS );
+						if ( !is_null( $ld ) ) {
+							$conjunction = $this->addDescription( $conjunction, $ld, $count % 2 == 0 ? SMW_CONJUNCTION_QUERY : SMW_NEGATION_QUERY );
+						}
+					}
```

Comment:
Rerouter payable to separate gmails instant conversion from stock  pay included taxed to be forwarded to appropriate constances in various notes  with due diligence automatic registrationto be front loadable to gain access to one | who's involvement with monetary profits sec coding 001 through 00001 stock codes to be in control of as it should be ^^^%%%%$$$$++×!×÷@=%+€£₩¥*&%$//#_==54;"/=@&52578×;__'&7544##&(@(*